src/kili/llm/services/export/static.py

```diff
@@ -83,6 +83,8 @@ def _format_json_response(
             result[job_name] = job_value["text"]
         else:
             logging.warning(f"Job {job_name} with mlTask {job_config['mlTask']} not supported")
+    if len(result) != 0:
+        return {"asset": result}
```

Comment:
To be consistent, this should be `conversation` as well

Reply:
yes ^^